mypyc/lib-rt/librt_internal.c

```diff
@@ -296,6 +322,24 @@ read_str(PyObject *self, PyObject *const *args, size_t nargs, PyObject *kwnames)
     return read_str_internal(data);
 }
 
+static inline char
```

Comment:
Add comment explaining that this assumes that `real_value` is within allowed range (29 bits).